release/release.md

```diff
@@ -0,0 +1,122 @@
+# How to do a Kata Containers Release
+
+* [Introduction](#Introduction)
+* [Requirements](#Requirements)
+* [Release process](#release-process)
+
+# Introduction
+
+If you are reading this document, you might want to create a Kata Containers
+Release.
+
+The Kata Containers Release Process is defined in the following
+[document][release-process-definition]. To simplify this process, we have
+created a [Release Checklist][release-checklist]. Finally, to simplify the
+Release Checklist process we have automated most of the process, this document
+guides on how to use release scripts instead of do all the checklist manually.
+
+
+## Requirements
+
+- It is recommended to have at least 2 GB of free disk space to perform these tasks.
+
+- [Go](https://golang.org/doc/install#)
+
+- [hub](https://github.com/github/hub)
+  If you use GitHub 2FA, configure the hub manually to create a file ~/.config/hub
+  ```yaml
+  github.com:
+  - user: YOUR_USER
+  oauth_token: TOKEN
+  protocol: https
+  ```
+- OBS account with permissions on /home:katacontainers (https://build.opensuse.org/project/subprojects/home:katacontainers)
+
+- GitHub permissions to push tags and creates Releases in Kata repositories.
```

Comment:
Do you need to be a github project "owner" for that? I suspect so. Might be worth stating if so.

Reply:
I am not sure about it, I think we need a group with push tag permissions, @jodh-intel can we create one for this? lets name it "release" if is possible to limit only to push tags better :)

Reply:
I'm not sure how to create a group that only has tag perms. Anyone else?

Reply:
I'm not sure you can. The groups don't have a very fine grained perm setting iirc. The tokens have a finer grain, but still not that fine grained I think.